packages/cli/src/commands/rolling-release/index.ts

```diff
@@ -22,13 +24,37 @@ export default async (client: Client): Promise<number> => {
   });
   let parsedArguments = null;
 
-  const flagsSpecification = getFlagsSpecification(
-    rollingReleaseCommand.options
+  // Get the subcommand, accounting for both full command name and aliases
+  const args = client.argv.slice(2);
+  const firstArg = args[0];
+
+  // Check if first arg is the command name or an alias
+  const isCommandOrAlias = firstArg === 'rolling-release' || firstArg === 'rr';
```

Comment:
Looks like this should include the `release` alias:
```suggestion
  const isCommandOrAlias = firstArg === 'rolling-release' || firstArg === 'rr' || firstArg === 'release';
```

Also, maybe the `getSubcommand` utility would come in handy here? Looks like this is used in other commands with subcommands: https://github.com/vercel/vercel/blob/3fedd2a6ea794032bc2c102808b0077f06548e5a/packages/cli/src/commands/alias/index.ts#L46